src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs

```diff
@@ -667,21 +667,18 @@ public void ExecuteAdditionalFileActions(
         private void ExecuteSyntaxNodeAction<TLanguageKindEnum>(
             SyntaxNodeAnalyzerAction<TLanguageKindEnum> syntaxNodeAction,
             SyntaxNode node,
-            ISymbol containingSymbol,
-            SemanticModel semanticModel,
+            ExecutionData executionData,
```

Comment:
nit: Consider passing by ref readonly?

Reply:
is that necessarily better?  all the values were being passed by value before.  i can certainly do that.  though then i assume we'd want to do that in the lambda callbacks as well.  And that will be enormously painful *until* we can use the language feature of being able to use 'in' without specifying the full type :)

Reply:
I didn't think of the `in` without type aspect. We can leave as-is. It's not worse than today.